packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Http/Client/Structure/MultiClient/MultiClientTests.cs

```diff
@@ -3,8 +3,8 @@
 
 using System.Linq;
 using System.Threading.Tasks;
-using Client.Structure.Service;
-using Client.Structure.Service.Models;
+using Client.Structure.Service.multi.client;
```

Comment:
is this expected?

Reply:
Yes, we renamed the yaml files of the folders within structure because we had naming conflicts with Client.Structure.Service.